BZ 1906890 - Documentation refers to append-bootstrap.ign but the example of that file has been removed from the documentation (#28059)
openshift/openshift-docs: modules/installation-vsphere-machines.adoc

+
*** Optional: In the event of cluster performance issues, from the *Latency Sensitivity* list, select *High*.
*** Click *Edit Configuration*, and on the *Configuration Parameters* window, click *Add Configuration Params*. Define the following parameter names and values:

> **** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded Ignition config file for this machine type.  Note: For the bootstrap node, ignition must be provided in `guestinfo.ignition.config.data` in the *Configuration Parameters* window.  This is due to a restriction in the maximum size of data which may be provided in a vApp property.

Reviewer: ```suggestion
**** `guestinfo.ignition.config.data`: Paste the contents of the base64-encoded Ignition config file for this machine type. Note for the bootstrap node, the Ignition config file must be provided in `guestinfo.ignition.config.data` in the *Configuration Parameters* window. This is due to a restriction in the maximum size of data that can be provided in a vApp property.
```

Reviewer: @rvanderp3 just some wordsmithing nits remaining for me, then it'll be good from my side. Thanks!

Author: No problem at all

Author: Done!

Reviewer: Thank you!

**** Navigate to a virtual machine from the vCenter Server inventory.
**** On the *Configure* tab, expand *Settings* and select *vApp options.*
**** Scroll down and under *Properties*, apply the configurations that you just edited.

> **** Note: For the bootstrap node, ignition must be provided in `guestinfo.ignition.config.data` in the *Configuration Parameters* window.  This is due to a restriction in the maximum size of data which may be provided in a vApp property.

Reviewer: @rvanderp3 does it make sense to combine this info with the sub-bullet several lines up?
>`guestinfo.ignition.config.data`: Paste the contents of the base64-encoded Ignition config file for this machine type.

Author: yep, I think it would be clarifying to move that there.  Making that edit.
